src/main/scala/geotrellis/sdg/PopulationNearRoads.scala

```diff
@@ -80,32 +83,42 @@ object PopulationNearRoads extends CommandApp(
 
         val result: Map[Country, PopulationSummary] =  {
           val future = Future.traverse(countries)( country => Future {
-            spark.sparkContext.setJobGroup(country.code, country.name)
-
             val rasterSource = country.rasterSource
             println(s"Reading: $country: ${rasterSource.name}")
+
             val layout = LayoutDefinition(rasterSource.gridExtent, 256)
 
-            val job = new PopulationNearRoadsJob(country, grumpRdd, layout, LatLng,
-              { t => t.isPossiblyMotorRoad /* && t.isStrictlyAllWeather*/ } )
+            spark.sparkContext.setJobGroup(country.code, country.name)
+
+            val job: PopulationNearRoadsJob = country match {
+              case cl: CountryLookup =>
+                new PopulationNearRoadsLookupJob(cl, grumpRdd, layout, LatLng,
+                  { t => t.isPossiblyMotorRoad /* && t.isStrictlyAllWeather*/ } )
+              case cp: CountryProvided =>
+                new PopulationNearRoadsProvidedJob(cp, layout, LatLng)
+            }
+
+            job.persist()
 
-            job.grumpMaskRdd.persist(StorageLevel.MEMORY_AND_DISK_SER)
```

Comment:
Why were these (and their equivalent `unpersist` calls) removed?

Reply:
EDIT: After cruising a bit further into the review, saw the `persist()` and `unpersist()` methods on the new trait.